include/triton/backend/backend_input_collector.h

```diff
@@ -62,6 +62,12 @@ class BackendInputCollector {
 
   ~BackendInputCollector();
 
+  // Return whether the entire input is in a contiguous buffer. If returns true,
+  // the properties of the contiguous input buffer will also be returned.
+  bool GetInputBufferIfContiguous(
```

Comment:
Exposing this will require changes in backends code. Please look at my suggestion [here ](https://github.com/triton-inference-server/openvino_backend/pull/20) and let me know what you think.